packages/document-grounding/src/spec/api.yaml

```diff
@@ -0,0 +1,1494 @@
+openapi: 3.0.0
+info:
+  title: Document Grounding Pipeline API
+  version: 0.1.0
+  description: "SAP AI Core - API Specification AI Data Management api's"
+paths:
+  '/lm/document-grounding/pipelines':
+    post:
+      operationId: pipeline.v1.pipeline_endpoints.create_pipeline
+      description: 'Create a pipeline'
+      tags:
+        - pipelines
+      parameters:
+        - in: header
+          name: AI-Resource-Group
+          required: true
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/createPipeline'
+      responses:
+        '201':
+          description: Returns pipelineId on successful creation.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/pipelineId'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+    get:
+      parameters:
+        - in: header
+          name: AI-Resource-Group
+          required: true
+          schema:
+            type: string
+        - $ref: '#/components/parameters/top'
+        - $ref: '#/components/parameters/skip'
+        - $ref: '#/components/parameters/count'
+      operationId: pipeline.v1.pipeline_endpoints.get_all_pipeline
+      description: 'Get all pipelines'
+      tags:
+        - pipelines
+      responses:
+        '200':
+          description: Returns all pipelines for the tenant.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/getPipelines'
```

Comment:
@jjtang1985 This schema name should get rid of `get` at the beginning. This makes the generated type also called `GetPipelines` instead of `Pipelines`

Reply:
Already communicated